zoom_monitor.py

```diff
@@ -257,12 +261,7 @@ def count_hands(self):
         """
         Return the number of people in the participants list with raised hands
         """
-        # If the meeting has ended, we can't count the hands any more, so raise
-        # a MeetingEndedException.
         self._checkIfMeetingEnded()
-
-        # If someone starts recording the meeting, we'll get a pop-up modal
-        # warning us about that before we can count hands again.
         self._acknowledge_recording()
```

Comment:
Without the comment, this function needs a better name (as written, I wonder what recording we're acknowledging). Maybe `_ignore_recording` or `_maybe acknowledge_recording`, or a shorter comment about how we don't care if someone starts recording the meeting?

Reply:
I like `_ignore_recording`!

Reply:
A symptom of bad function names is that you need to add comments to explain what the names were supposed to mean. :smirk: Thanks for cleaning up my mess!